apriltag_ros/src/continuous_detector.cpp

```diff
@@ -195,16 +202,22 @@ void ContinuousDetector::imageCallback (
   // compute the pixel values for the FOV pixel buffer (ie. resizing the FOV)
   fov_pixel_buffer_height_ = (camera_info->height/2) - ((camera_info->height * fov_size_scaler_) / 2);
   fov_pixel_buffer_width_ = (camera_info->width/2) - ((camera_info->width * fov_size_scaler_) / 2);
+  bool detector_validations_failed{ false };
 
   // Check if not in fov or not in detection range and publish an empty detection message
   if (!isTagInFOV(tag_pose_) || !isTagInDetectionRange(tag_pose_))
   {
-    // ROS_INFO_STREAM_THROTTLE(1.0, "FOV or range check not passed");
-    publishEmptyDetection(image_rect->header);
-    return;
+    if (detection_timeout_)
+    {
+      // ROS_INFO_STREAM_THROTTLE(1.0, "FOV or range check not passed");
```

Comment:
```suggestion
      ROS_DEBUG_STREAM_THROTTLE(1.0, "FOV or range check not passed");
```
:thinking: or maybe it should actully be INFO, or even WARN :thinking:

Reply:
no, I'll remove